src/main/groovy/edu/oregonstate/mist/api/Resource.groovy

```diff
@@ -22,6 +22,11 @@ abstract class Resource {
      */
     public static final Integer DEFAULT_PAGE_SIZE = 10
 
+    /**
+     * Default max page size used in pagination.
+     */
+    public static Integer MAX_PAGE_SIZE = 10000
```

Comment:
As we discussed, the queries aren't considering pagination and page[size] is being ignored.